editor/src/marionette/pose-graph-editor-api.ts

```diff
@@ -92,6 +93,17 @@ export function getInputDefaultDisplayName(inputKey: poseGraphOp.InputKey) {
     }
 }
 
+export function getPoseGraphNodeInputAttrs(node: PoseGraphNode, inputKey: poseGraphOp.InputKey) {
+    const [propertyName] = inputKey;
+    const attrs = attr(node.constructor, propertyName);
+    delete attrs.type;
+    delete attrs.ctor;
```

Comment:
An input's type is not specified by cc-class system. It's specified by `@poseGraphType`